### ISSUE TITLE
{ selenider_elements (16) } to vector

### ISSUE THREAD

**Issue by nipnipj** [ashbythorpe/selenider#36]
Can we transform and `{ selenider_elements (16) }` object to a col vector `c()`

**Comment by ashbythorpe** [comment]
No, you can't. You can use functions like `elem_filter()`, `elem_find()`, `[`/`[[`, `find_each_element()` and `find_all_elements()` to make further sub-selections. You can also convert it to a list using `as.list()`.

See https://ashbythorpe.github.io/selenider/dev/articles/advanced-usage.html#element-collections for more advanced info on how they work.
